modules/openapi-generator/src/test/java/org/openapitools/codegen/java/micronaut/JavaMicronautServerCodegenTest.java

```diff
@@ -487,4 +486,46 @@ public void doRepeatOperationForAllTags() {
             .hasAnnotation("JacksonXmlProperty", Map.of("localName", "\"item\""))
             .hasAnnotation("JacksonXmlElementWrapper", Map.of("localName", "\"activities-array\""));
     }
+
+    @Test
+    public void testMultipleContentTypesToPath() {
```

Comment:
The new unit test here and in JavaMicronautClientCodegenTest both fail with a NullPointerException when I run them. Is it different for you?

Reply:
fixed